app/src/main/java/com/cortado/minionify/accumulators/exceptions/KeyDoesNotExists.java

```diff
@@ -0,0 +1,7 @@
+package com.cortado.minionify.accumulators.exceptions;
+
+public class KeyDoesNotExists extends RuntimeException {
+    public KeyDoesNotExists() {
```

Comment:
no reason to declare an empty constructor

Reply:
Done.